scripts/crons/cron_libraries.sh

```diff
@@ -47,53 +46,29 @@ techo "Processing for DATE: $DATE and WEEK: $WEEK"
 techo "Phase 1: start"
 for I in Firefox Thunderbird SeaMonkey
 do
-  techo "Phase 1: Product: $I"
-  techo "Running psql query for version list."
-  VERSIONS=`psql -t -U $databaseUserName -h $databaseHost $databaseName -c "select version, count(*) as counts from reports_${WEEK}  where completed_datetime < $SQL_DATE and completed_datetime > ($SQL_DATE - interval '24 hours') and product = '${I}' group by version order by counts desc limit 3" | awk '{print $1}'`
-  for J in $VERSIONS
-  do
-    techo "Phase 1: Version: $J start"
-    techo "Pulling processed JSON from PostgreSQL"
-    psql -t -U $databaseUserName -h $databaseHost $databaseName -c "copy (select processed_crashes.uuid, processed_crash from processed_crashes join reports_${WEEK} on (reports_${WEEK}.uuid::uuid = processed_crashes.uuid) where completed_datetime < $SQL_DATE and completed_datetime > ($SQL_DATE - interval '24 hours') and product = '${I}' and version = '${J}') to stdout with csv quote e'\x01' delimiter e'\x02'" | /home/socorro/tar_crashes.py $TMPDIR/${I}_${J}.tar
-    techo "per-crash-core-count.py > $TMPDIR/${DATE}_${I}_${J}-core-counts.txt"
-    $PYTHON /data/crash-data-tools/per-crash-core-count.py -p ${I} -r ${J} -f $TMPDIR/${I}_${J}.tar > $TMPDIR/${DATE}_${I}_${J}-core-counts.txt
-    techo "per-crash-interesting-modules.py > $TMPDIR/${DATE}_${I}_${J}-interesting-modules.txt"
-    $PYTHON /data/crash-data-tools/per-crash-interesting-modules.py -p ${I} -r ${J} -f $TMPDIR/${I}_${J}.tar > $TMPDIR/${DATE}_${I}_${J}-interesting-modules.txt
-    techo "per-crash-interesting-modules.py > $TMPDIR/${DATE}_${I}_${J}-interesting-modules-with-versions.txt"
-    $PYTHON /data/crash-data-tools/per-crash-interesting-modules.py -v -p ${I} -r ${J} -f $TMPDIR/${I}_${J}.tar > $TMPDIR/${DATE}_${I}_${J}-interesting-modules-with-versions.txt
-    techo "per-crash-interesting-modules.py > $TMPDIR/${DATE}_${I}_${J}-interesting-addons.txt"
-    $PYTHON /data/crash-data-tools/per-crash-interesting-modules.py -a -p ${I} -r ${J} -f $TMPDIR/${I}_${J}.tar > $TMPDIR/${DATE}_${I}_${J}-interesting-addons.txt
-    techo "per-crash-interesting-modules.py > $TMPDIR/${DATE}_${I}_${J}-interesting-addons-with-versions.txt"
-    $PYTHON /data/crash-data-tools/per-crash-interesting-modules.py -v -a -p ${I} -r ${J} -f $TMPDIR/${I}_${J}.tar > $TMPDIR/${DATE}_${I}_${J}-interesting-addons-with-versions.txt
-    techo "Phase 1: Version: $J end"
-  done
-  techo "Phase 1: end"
+    techo "Phase 1: Product: $I"
+    techo "Running psql query for version list."
+  VERSIONS=`psql -t -U $SECRETS_POSTGRESQL_DATABASE_USERNAME -h $RESOURCE_POSTGRESQL_DATABASE_HOSTNAME breakpad -c "select version_string from product_versions where product_name = '${I}' and '${SQL_DATE}' between build_date and sunset_date" | awk '{print $1}'`
```

Comment:
I asked KaiRo in IRC to confirm how versions are chosen. I can't remember if its all active versions or featured versions. If you've already done this bit, please ignore this comment.